core/src/main/java/org/elasticsearch/cluster/metadata/MetaData.java

```diff
@@ -915,6 +946,29 @@ public Builder customs(ImmutableOpenMap<String, Custom> customs) {
             return this;
         }
 
+        public Builder put(final IndexTombstone tombstone) {
```

Comment:
I think we should just have one way of passing this? all the syntactic sugar is unneedd?